### ISSUE TITLE
Not working in Windows

### ISSUE THREAD

**Issue by Arjun-NA** [jgust/SublimeCscope#2]
Hi @jgust 

Thanks for trying but, Not able to even download with the command you have posted. 

I edited the command to 
`` git clone https://github.com/jgust/SublimeCscope.git "%APPDATA%/Sublime Text/SublimeCscope"  ``
But still doesnt, so I understood it have to be copied to Packages folder.
i.e
`` git clone https://github.com/jgust/SublimeCscope.git "%APPDATA%/Sublime Text/Packages/SublimeCscope"  ``
It shows for a second and sublime throws it out ( deletes that folder )

Thanks,
Arjun

**Comment by jgust** [comment]
Hello @Arjun-NA 

Sorry for the sluggish reply.  Just came back from vacation.

You are correct, It should be the Packages folder. It's a typo in the instructions.

The reason it is getting deleted directly is due to a relatively new feature in Package Control, that always removes any unknown packages it encounters.  The workaround is:

1. Make sure the sublime-text application is closed.
2. clone the package: 
`git clone https://github.com/jgust/SublimeCscope.git "%APPDATA%/Sublime Text/Packages/SublimeCscope"`
3. Add `SublimeCscope` to the list of installed packages in `"%APPDATA%/Sublime Text/Packages/User/Package Control.sublime-settings"` using a different text editor
4. start sublime-text

I just recently started using windows as a secondary development environment, so this packages hasn't been tested much on windows yet. It seems to work fine, but one annoying thing I've noticed is that each time `cscope` is called by the plugin it is run in a new terminal windows that pops up and closes again.  I will have to fix that once I get the time.

Cheers,
Julius
